src/util.ts

```diff
@@ -627,3 +629,30 @@ export async function normalizeAndVerifySourceDir(sourceDir: string): Promise<st
 export function isCodespaces(): boolean {
   return !!process.env["CODESPACES"];
 }
+
+export function getAllFilePaths(dir: vscode.Uri, filename: string): string[] | undefined {
+  const regex: RegExp = new RegExp(`(\\/|\\\\)${escapeStringRegExp(filename)}$`);
```

Comment:
It's unclear to me why we need the `(\\/|\\\\)` as part of the regular expression.  Can you comment this, or remove it if it's not necessary?

Reply:
`(\\/|\\\\)` and `(\/|\\\\)`  work similar !

having that checks for having `\\` or `/` before string `CMakeLists.txt` so that these paths don't get matched:
`D:\\project\CMakeLists.txt` or `D:\\project\\_CMakeLists.txt`